helmfile/values/podsecuritypolicies/common/fluentd.yaml.gotmpl

```diff
@@ -50,4 +50,8 @@ constraints:
           - hostPath
           - projected
   {{- end }}
+mutations:
+  exceptions:
+    - app.kubernetes.io/instance: fluentd
+    - app.kubernetes.io/instance: fluentd-forwarder
```

Comment:
I've been testing this a bit and do you think it would be better to have an implicit removal of `drop: [ALL]` if we set allowed capabilities, or to add a per service mutation exception to skip adding `drop: [ALL]`? 
(I would want to avoid setting wide exceptions, though it is better now that these are limited to certain namespaces.)

Reply:
I first thought of implicit removal if one has configured for example `allow.allowedCapabilities`, but now I'm leaning towards the option of disabling mutations per service level.